pages/common/find.md

```diff
@@ -7,18 +7,14 @@
 
 `find {{root_path}} -name '{{*.ext}}'`
 
-- Find files by matching multiple patterns:
+- Find files matching matching multiple path/name patterns:
```

Comment:
```suggestion
- Find files matching multiple path/name patterns:
```
That's too many `matching`s 😄

Reply:
it should be 
```suggestion
- Find files by matching multiple path/name patterns:
```

Reply:
I prefer the first suggestion.  @CleanMachine1 while technically correct, your suggestion makes it sound as though we were talking about _how_ `find` internally works for this invocation.

Reply:
This is a funny typo, and I'm surprised I didn't notice! 🙂

Reply:
I'm surprised we all didn't notice 😄